### PR TITLE
added extra validation in domain details form to block "glue" records

### PR DESCRIPTION
**Pull request by vesellov**

### CODE REVIEW

File: src/front/views.py [hunk 1]
```diff
@@ -134,6 +135,11 @@ def form_valid(self, form):
 
         domain_obj = form.save(commit=False)
 
+        for nameserver in domain_obj.list_nameservers():
```
Comment (ilhanmercan) [inline src/front/views.py:138]: why is this needed? we have already in form validation error which returns exactly this error. i think that should be enough

Comment (vesellov) [reply src/front/views.py:138]: when we create a new domain - the record in DB not yet exists, but form code & validation already triggered...  so `self.instance.name` is empty in the form code and so hostnames can't be verified against the domain name....   I didn't found a better solution and placed same validation in the view as well...  if you know a better approach - lets do it!

Comment (ilhanmercan) [reply src/front/views.py:138]: okay, it makes now. thanks for explaining

**Review by ilhanmercan — APPROVED** [review]
